rolling/distribution.yaml

```diff
@@ -4430,5 +4430,16 @@ repositories:
       url: https://github.com/ros2/yaml_cpp_vendor.git
       version: master
     status: maintained
+  zenoh-bridge-dds:
+    release:
```

Comment:
@JEnoch no `doc` entry?

Reply:
@hidmic I was following the [bloom 1st release tuto](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) that doesn't refer to the doc. I was surprised when `bloom-release` asked for. But as it proposes a `y/n` answer, I assumed the doc was not required for a 1st release and thought to push it later in another PR.
The doc content is ready as it will be a copy of this [README.md](https://github.com/eclipse-zenoh/zenoh-plugin-dds#readme). I just need to check if I have to manually copy/adapt it for ROS2 doc, or if I can automatize this.

Reply:
@hidmic: I just pushed a commit adding the `doc` entry.
Can you please correct me if I'm misunderstood those points:
 - the [doc indexing tutorial](https://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation) only refers to ROS packages and is obsolete for ROS2, right ? If I look at http://wiki.ros.org/turtlesim it lists only ROS distros. While looking at https://index.ros.org/p/turtlesim/ it lists only to ROS2 distros
 - As a consequence there is no longer need to [create a wiki page](https://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation#Create_a_wiki_page_for_your_package_in_very_short_steps). The package page at index.ros.org will copy the README.md from the `doc` repo, correct ?